AFQ/utils/stats.py

```diff
@@ -0,0 +1,7 @@
+def contrast_index(x1, x2):
+    """
+    Calculate the contrast index between two arrays.
+    The contrast index is symmetrical with respect to
+    the choice of the baseline for comparison
+    """
+    return (x1 - x2) / (x1 + x2)
```

Comment:
To answer your question about this: I have not seen this implemented elsewhere. This is a good place to put it! But please add a docstring and some unit testing, please.